<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionConverterDialog</class>
 <widget class="QDialog" name="PositionConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>坐标转换工具</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="copyPushButton">
     <property name="text">
      <string>复制结果</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="conversionLabel">
     <property name="text">
      <string>转换</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="conversionComboBox">
     <item>
      <property name="text">
       <string>LL 转 MGRS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MGRS 转 LL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QStackedWidget" name="PositionStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="LLToMGRSPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="UtQtLatPosLineEdit" name="latitudeLineEdit">
         <property name="placeholderText">
          <string>0n</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="resultMGRSLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="resultMGRSLabel">
         <property name="text">
          <string>结果(MGRS):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="longitudeLabel">
         <property name="text">
          <string>经度:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="positionLLALabel">
         <property name="text">
          <string>位置(LL):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="UtQtLonPosLineEdit" name="longitudeLineEdit">
         <property name="placeholderText">
          <string>0e</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="latitudeLabel">
         <property name="text">
          <string>纬度:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MGRSToLLPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="positionMGRSLabel">
         <property name="text">
          <string>位置(MGRS):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="resultLLLabel">
         <property name="text">
          <string>结果(LL):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="MGRSLineEdit">
         <property name="placeholderText">
          <string>(e.g. 31NAA6602100000)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="resultLLLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtLatPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLonPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>conversionComboBox</tabstop>
  <tabstop>latitudeLineEdit</tabstop>
  <tabstop>longitudeLineEdit</tabstop>
  <tabstop>resultMGRSLineEdit</tabstop>
  <tabstop>MGRSLineEdit</tabstop>
  <tabstop>resultLLLineEdit</tabstop>
  <tabstop>copyPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
